accelerometer/tests/check_data_quality_outputs.py

```diff
@@ -0,0 +1,13 @@
+import json
```

Comment:
Also where are we calling this script to check the output? Maybe it would be easier if we also have a reference json file instead and use the `tianhaoz95/mirror-action@v1.0.1` for comparison?

Reply:
This is meant to have disappeared, will delete!